TWLight/users/tests.py

```diff
@@ -357,162 +357,6 @@ def test_sent_application(self):
         app.refresh_from_db()
         self.assertNotIn("Withdraw", response.render().content.decode("utf-8"))
 
-    def test_my_library_authorizations_and_collections(self):
```

Comment:
It looks like we may be losing some authorization test coverage here? Maybe instead of dropping this entirely, you could revert the change I made here:
https://github.com/WikipediaLibrary/TWLight/commit/160fcee59bb97eecc696e7d66082c942019ed23a#diff-bdbc11d651f956ef2d399ac5ec4d943b6ede2018b9df52b37b465a60e8752e8e

So that this test is really only concerned with the authorizations themselves.

This was just based on a read through, so I could be mistaken. Happy to circle back around for another look on Tuesday.

Reply:
I think the change you added to the test is the only thing we should keep, since we don't make distinctions between manual authorizations and bundle authorizations anymore.

Reply:
This was a red herring in my part. As you say, we don't care about some of the differences in the auths anymore.